importer/PhEn.py

```diff
@@ -0,0 +1,102 @@
+from Monument import Monument, Dataset
+import importer_utils as utils
+import importer as importer
+
+
+class PhEn(Monument):
+
+    def update_labels(self):
+        english = utils.remove_markup(self.site_name)
+        self.add_label("en", english)
+
+    def update_descriptions(self):
+        english = "Declared Cultural Property in the Philippines"
+        self.add_description("en", english)
+
+    def set_location(self):
+        """Set location based on linked article."""
+        loc_q = None
+        if self.has_non_empty_attribute("location"):
+            if utils.count_wikilinks(self.location) == 1:
+                loc_q = utils.q_from_first_wikilink("en", self.location)
+
+            if loc_q:
+                self.add_statement("location", loc_q)
+            else:
+                self.add_to_report("location", self.location, "location")
+
+    def set_adm_location(self):
+        """
+        Set the administrative location.
+
+        First, try getting it via the wikilinked 'province'
+        and check that it's present in the province mapping.
+        If that fails, get the larger region via 'region-iso'
+        and mapping.
+        """
+        adm_q = None
+        prov_dic = self.data_files["provinces"]
+        reg_dic = self.data_files["regions"]
+
+        if utils.count_wikilinks(self.province) == 1:
+            adm_q_try = utils.q_from_first_wikilink("en", self.province)
+            if adm_q_try in [x["item"] for x in prov_dic]:
+                adm_q = adm_q_try
+
+        if adm_q is None:
+            iso = self.region_iso
+            iso_match = utils.get_item_from_dict_by_key(reg_dic,
+                                                        search_term=iso,
+                                                        search_in="iso")
+            if len(iso_match) == 1:
+                adm_q = iso_match[0]
+
+        if adm_q:
+            self.add_statement("located_adm", adm_q)
+        else:
+            self.add_to_report("province", self.province, "located_adm")
+
+    def set_wd_item_from_name(self):
```

Comment:
It might be safer to here do:
```python
if utils.count_wikilinks(self.site_name) == 1:
    wikilink = utils.get_wikilinks(text)[0]
    self.monument_article = wikilink.title
```
That way you may then reuse the normal logic of `exists_with_monument_article()` (including section handling and blacklist integration)